src/rtc.h

```diff
@@ -116,6 +116,11 @@ typedef void(*voidCallbackPtr)(void *);
 #define RTC_Alarm_IRQHandler RTC_TAMP_IRQHandler
 #endif
 
+#if defined(STM32L5xx)
+#define RTC_Alarm_IRQn RTC_IRQn
+#define RTC_Alarm_IRQHandler RTC_IRQHandler
+#endif
+
```

Comment:
Simply add  ` || defined(STM32L5xx)` line [110](https://github.com/stm32duino/STM32RTC/pull/44/files#diff-789875f7be63ae7be40fecc87506ddc13a6409292f38ab201896982d20010f59R110)

And remove those lines:
```suggestion
```